src/app/shared/menu/dropdown.css

```diff
@@ -0,0 +1,59 @@
+.dropdown {
```

Comment:
the custom-date-range-dropdown.component.css will instead be using this css with a few custom settings with ticket #123 when the full pop up is implemented that incorporates this standard range list and intervals along with the date range into an advanced menu